modules/account-lib/src/coin/baseCoin/iface.ts

```diff
@@ -120,3 +120,13 @@ export interface TransactionExplanation {
   changeAmount: string;
   fee: TransactionFee;
 }
+
+/**
+ * Fee options.
+ */
+export type FeeOptions = {
```

Comment:
these fee options are meant to be standardized.

it was previously defined in Dot, but i moved it out here so it can be re-used for multiple coins.
original comment was: `Fee options based on the BitgoJS standard doc.`